src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -209,8 +209,8 @@ export const updateDonorsCompletionStats = async (
 	const exceptionsCache = createMissingEntityExceptionCache();
 
 	// clone each donor
-	const updatedDonorPromises = donors.map((donor) =>
-		updateSingleDonorCompletionStats(donor, exceptionsCache),
+	const updatedDonorPromises = donors.map(
+		async (donor) => await updateSingleDonorCompletionStats(donor, exceptionsCache),
 	);
```

Comment:
not needed and potentially confusing.
asyncs in map will return arrays of promises.